apps/datahub/src/app/router/datahub-router.service.ts

```diff
@@ -29,16 +29,21 @@ export class DatahubRouterService {
     return [
       {
         path: '',
-        redirectTo: `${ROUTER_ROUTE_HOME}/${ROUTER_ROUTE_NEWS}`,
+        redirectTo: ROUTER_ROUTE_NEWS,
         pathMatch: 'full',
       },
       {
         path: ROUTER_ROUTE_HOME,
-        redirectTo: `${ROUTER_ROUTE_HOME}/${ROUTER_ROUTE_SEARCH}`,
+        redirectTo: ROUTER_ROUTE_NEWS,
         pathMatch: 'full',
       },
       {
         path: ROUTER_ROUTE_HOME,
+        redirectTo: ``,
+        pathMatch: 'prefix',
+      },
+      {
+        path: '',
```

Comment:
Does it work if you just have one config for `''` path instead of 2 like here ?

Reply:
It does'nt. If I remove the 'full' one, slash path is broken, and if I remove the 'prefix' it's new/search/org which is broken.